widgetssdk/src/main/java/com/glia/widgets/engagement/EngagementRepositoryImpl.kt

```diff
@@ -333,6 +333,7 @@ internal class EngagementRepositoryImpl(private val core: GliaCore, private val
         currentEngagement?.also {
             unsubscribeFromEvents(it)
             currentEngagement = null
+            resetState()
```

Comment:
Could you please add the following to both omnicore and callVisualizer engagements?

Reply:
done